libsubid/api.h

```diff
@@ -1,9 +1,8 @@
 #include "subid.h"
 #include <stdbool.h>
 
-int get_subuid_ranges(const char *owner, struct subordinate_range ***ranges);
-int get_subgid_ranges(const char *owner, struct subordinate_range ***ranges);
-void subid_free_ranges(struct subordinate_range **ranges, int count);
```

Comment:
Well, a helper still would be handy.
While we can free `ranges` in one shot (no need to free every `subordinate_range` element), we still need to iterate over every array item first to free `subordinate_range::owner`...

Reply:
Eh I should have left it as an array of poitners to subordinate_range.

Reply:
That's what I meant when wrote "this seems to be deeply embedded into existing code..." :)